pkg/services/ngalert/api/tooling/Makefile

```diff
@@ -32,7 +32,9 @@ api.json: spec-stable.json
 	go run cmd/clean-swagger/main.go -if $(<) -of $@
 
 swagger-codegen-api:
-	docker run --rm -v $$(pwd):/local --user $$(id -u):$$(id -g) parsertongue/swagger-codegen-cli:3.0.32 generate \
+	docker run --rm \
+	--platform linux/amd64 \
```

Comment:
This is needed to run the image on M1 macs under emulation. I was getting errors when running it without the flag.